lnmuxrpc/lnmux.proto

```diff
@@ -73,3 +77,23 @@ message CancelInvoiceRequest {
 }
 
 message CancelInvoiceResponse {}
+
+message ListInvoiceRequest {
+    uint64 num_max_invoices = 1;
+    uint64 index_offset = 2;
+}
+
+message ListInvoiceResponse {
+    repeated Invoice invoice = 1;
+    uint64 first_index_offset = 2;
```

Comment:
Isn't this just the request index passed back?

Reply:
I tried to answer that here:
https://github.com/bottlepay/lnmux/pull/104#discussion_r1009174857